packages/widget-chat/src/actions/conversation.js

```diff
@@ -28,8 +45,21 @@ export function createConversationWithUser(userId, spark) {
     spark.conversation.create({
       participants: [userId]
     }, {
-      activitiesLimit: 10
+      latestActivity: true,
+      activitiesLimit: 30
```

Comment:
That said, export action names from their files seems perfectly reasonable to me.